openlibrary/plugins/upstream/addbook.py

```diff
@@ -173,16 +184,24 @@ def POST(self):
             return self.no_match(saveutil, i)
 
     def find_matches(self, saveutil, i):
-        """Tries to find an edition or a work or multiple works that match the given input data.
+        """
+        Tries to find an edition or a work or multiple works that match the given input data.
 
         Case#1: No match. None is returned.
         Case#2: Work match but not editon. Work is returned.
         Case#3: Work match and edition match. Edition is returned
         Case#3A: Work match and multiple edition match. List of works is returned
```

Comment:
`Case#3A: Work match and multiple edition match. List of works is returned` does not make sense, what is this case, and what does it return?